apps/src/applab/DataTable.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * @overview Component for editing a data table.
+ */
+
+import { DataView } from './constants';
+import React from 'react';
+import { changeView } from './redux/data';
+import color from '../color';
+import { connect } from 'react-redux';
+
+const styles = {
+  link: {
+    color: color.purple,
+    fontFamily: "'Gotham 7r', sans-serif"
+  }
+};
+
+const DataTable = React.createClass({
+  propTypes: {
+    // from redux state
+    tableName: React.PropTypes.string.isRequired,
+    view: React.PropTypes.oneOf([DataView.OVERVIEW, DataView.PROPERTIES, DataView.TABLE]),
```

Comment:
I bet you could also do something like `oneOf(Object.keys(DataView))`. What you have might be better anyways tho, since it might be valuable to see the whole list again.